File: 3-Bimestre/Python/soma-numeros-pares.py
"""
Aluno: Carlos Eduardo Félix
3º Bimestre
Exercício 04 - Soma dos números pares
"""

numero = int(input("Digite um número inteiro positivo: "))
contador = 1
soma = 0

if numero > 0:
    while contador <= numero:
        if contador % 2 == 0:
            soma += contador
        contador += 1

    print("Soma dos pares:", soma)
else:
    print("Número inválido.")

File: 3-Bimestre/C++/soma-numeros-pares.cpp
/*
Aluno: Carlos Eduardo Félix
3º Bimestre
Exercício 04 - Soma dos números pares
*/
#include <iostream>
using namespace std;

int main() {
    int numero;
    int contador = 1;
    int soma = 0;

    cout << "Digite um numero inteiro positivo: ";
    cin >> numero;

    if (numero <= 0) {
        cout << "Numero invalido." << endl;
        return 0;
    }

    while (contador <= numero) {
        if (contador % 2 == 0) {
            soma += contador;
        }
        contador++;
    }

    cout << "Soma dos pares: " << soma << endl;
    return 0;
}
